openvino/conditional_compilation/include/openvino/cc/selective_build.h

```diff
@@ -233,9 +237,16 @@ bool match(char const *region, Ctx && ctx, T && val, Case && cs, Cases&&... case
 // Scope is enabled
 #define OV_CC_SCOPE_1(...) __VA_ARGS__
 
+#define OV_CC_DEFINE_0() 0
+#define OV_CC_DEFINE_1() 1
+
 #define OV_SCOPE(Module, region, ...)           \
     OV_CC_EXPAND(OV_CC_CAT(OV_CC_SCOPE_, OV_CC_SCOPE_IS_ENABLED(OV_CC_CAT3(Module, _, region)))(__VA_ARGS__))
 
+#define OV_SCOPE_DEFINE(Module, region)                                                     \
+    OV_CC_CAT(OV_CC_DEFINE_, OV_CC_SCOPE_IS_ENABLED(OV_CC_CAT3(Module, _, region)))()
```

Comment:
It can be replaced by:
#define OV_SCOPE_DEFINE(Module, region) OV_CC_CAT3(Module, _, region)

Usage:
#ifdef OV_SCOPE_DEFINE(Module, region)
    OV_ITT_SCOPED_TASK(domains::SIMPLE_Module, "region");
    cout << "Hello nGraph!";
#endif

Reply:
Changed `OV_SCOPE_SEMANTIC`